### PR TITLE
Implemented multiple tables and their relationships

### PR DESCRIPTION
**Pull request by vangelif**
# In this PR, I:

- Followed Gitflow principles.
- Documented in a professional way.
- Prepared complex queries.
- Used primary key & foreign key mechanism for joining tables.
- Took screenshots

### CODE REVIEW

File: data.sql [hunk 1]
```diff
@@ -53,4 +53,32 @@ WHERE NAME IN ('Charmander', 'Squirtle', 'Blossom');
...
+('William Tatcher', 45, '2000-04-23'),
+('Maisy Smith', 26, '2019-01-17'),
+('Stephanie Mendez', 64, '1981-05-04'),
+('Jack Harkness', 38, '2008-06-08');
+
+INSERT INTO visits(animal_id, vet_id, date) VALUES
+((SELECT id FROM animals WHERE name = 'Agumon'),(SELECT id FROM vets WHERE name = 'William Tatcher'), '2020-05-24'),
+((SELECT id FROM animals WHERE name = 'Agumon'),(SELECT id FROM vets WHERE name = 'Stephanie Mendez'), '2020-07-22'),
+((SELECT id FROM animals WHERE name = 'Gabumon'),(SELECT id FROM vets WHERE name = 'Jack Harkness'), '2021-02-02'),
+((SELECT id FROM animals WHERE name = 'Pikachu'),(SELECT id FROM vets WHERE name = 'Maisy Smith'), '2020-05-01'),
```
Comment (Davidosky007) [inline data.sql]: * Kindly insert the right date for this visit `Pikachu visited Maisy Smith on Jan 5th, 2020.` the `2020-05-01` is wrong

File: data.sql [hunk 2]
```diff
@@ -53,4 +53,32 @@ WHERE NAME IN ('Charmander', 'Squirtle', 'Blossom');
...
-SELECT * FROM animals;
\ No newline at end of file
+SELECT * FROM animals;
+
+INSERT INTO vets(name, age, date_of_graduation) VALUES
+('William Tatcher', 45, '2000-04-23'),
+('Maisy Smith', 26, '2019-01-17'),
+('Stephanie Mendez', 64, '1981-05-04'),
+('Jack Harkness', 38, '2008-06-08');
+
```
Comment (Davidosky007) [inline data.sql:63]: * Kindly insert the missing data below👇 
```
Insert the following data for specialties:
Vet William Tatcher is specialized in Pokemon.
Vet Stephanie Mendez is specialized in Digimon and Pokemon.
Vet Jack Harkness is specialized in Digimon.
```

**Review by Davidosky007 — CHANGES_REQUESTED** [review]
Hi @vangelif 👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
* Complex queries that answer analytical questions used properly 👍 
* Primary keys & foreign keys mechanism for joining tables used properly 👍 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @davidsoky007 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Zuhaib042 — APPROVED** [review]
Hi @vangelif 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
✅ Well done on implementing suggested changes
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**